frappe/utils/pdf.py

```diff
@@ -22,6 +22,33 @@
 	"RemoteHostClosedError",
 ]
 
+import sys
+import logging
+from logging.handlers import RotatingFileHandler
+class PDFLogger(object):
```

Comment:
No need to build custom logger here? You can use `frappe.logger` for this

Reply:
No, because the logger must behave like stdout

Reply:
```
		# wkhtmltopdf writes the pdf to stdout and errors to stderr
		# pdfkit v1.0.0 writes the pdf to file or returns it
		# stderr is written to sys.stdout if verbose=True is supplied
		# Set filename property to false, so no file is actually created
```